### DISCUSSION TITLE
Automated API documentation

### DISCUSSION THREAD

**Discussion by kangz12345** [snu-quiqcl/qiwis#69] [Ideas]
We would like to document the package automatically, since we are writing the docstrings.
As I said in https://github.com/snu-quiqcl/swift/discussions/55#discussioncomment-5260804, [sphinx.ext.napoleon](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html) seems a good candidate.

To use this, we need to learn how to use [sphinx](https://www.sphinx-doc.org/en/master/index.html).

**Comment by BECATRUE** [comment]
Wow.. It is amaizing! We need to learn it someday.
